rnd/autogpt_server/autogpt_server/server/server.py

```diff
@@ -314,153 +314,14 @@ async def websocket_router(self, websocket: WebSocket):
                     await autogpt_server.server.ws_api.handle_unsubscribe(
                         websocket, self.manager, message
                     )
-                elif message.method == Methods.EXECUTION_EVENT:
```

Comment:
Hu I thought I had already deleted all these!

Reply:
You did, in the template & versioning PR (#7376). I reverted it then because it had nothing to do with the rest of the PR. Yesterday I realized it was still there while working on #7739.